src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,36 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+	// iOS localization specific options
+    // TODO: refactor the code for DRY issues
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    // using NSNull comparison for JSON values set to null
+	if(command.arguments.count == 3 && command.arguments[2]) {
```

Comment:
There isn't much benefit from this outer `if` statement. Instead you can simply use `if-else` shorthand e.g. `NSDictionary *feedbackOptions = command.arguments.count > 2 ? command.arguments[2] : nil;`